JPS_Ontology/ontology/ontoreaction/OntoReaction.csv

```diff
@@ -66,9 +67,15 @@ indicatesUsageOf,Object Property,,,ReactionScale,InputChemical,exactly 1,A relat
 hasInputChemical,Object Property,,,ReactionExperiment,InputChemical,,A relation between a reaction experiment and its input chemicals. ,https://www.theworldavatar.com/kg/ontoreaction/,
 hasOutputChemical,Object Property,,,ReactionExperiment,OutputChemical,,A relation between a reaction experiment and its output chemicals. ,https://www.theworldavatar.com/kg/ontoreaction/,
 yieldLimitingSpecies,Object Property,,,Yield UNION Conversion,http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#Species,exactly 1,A relation between yield or conversion of a reaction experiment and the yield limiting species based on which the performance indicator was computed. ,https://www.theworldavatar.com/kg/ontoreaction/,
+isWrittenBy,Object Property,,,OperatorNotes,https://www.theworldavatar.com/kg/ontoprofile/Operator,,A relation between the notes for a reaction experiment and the operator that wrote it.,https://www.theworldavatar.com/kg/ontoreaction/,
+isOperatedBy,Object Property,,,ReactionExperiment,https://www.theworldavatar.com/kg/ontoprofile/Operator,,A relation between a reaction experiment and its operator.,https://www.theworldavatar.com/kg/ontoreaction/,
+hasOperatorNotes,Object Property,,,ReactionExperiment,OperatorNotes,,A relation between a reaction experiment and its notes written by the operator.,https://www.theworldavatar.com/kg/ontoreaction/,
+hasStartTime,Object Property,,,ReactionExperiment,http://www.w3.org/2006/time#Instant,,A relation between a reaction experiment and its start time.,https://www.theworldavatar.com/kg/ontoreaction/,
```

Comment:
The start and end times are automatically represented by the derived information framework (DIF) if DIF executes the experiment. What's the plan here?

Reply:
The start time and end time are not handled by any of our packages or libraries. The ontology is supposed to be passed on to a 3rd party where they use their own software to handle the recording of start time and end time.